### DISCUSSION TITLE
[Feature] Typing of person's name or location place in Explore will show results from said person or place (if in database)

### DISCUSSION THREAD

**Discussion by kikendo** [immich-app/immich#12394] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It's a bit strange that trying to do this does not yield results from within the database. The only way seems to be to go to "people" or "places" and click through

Please let us write these on the explore search bar and yield the relevant results.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Your request isn't entirely clear to me, can you give some more detail?

**Reply by kikendo** [reply]
Go to the search bar and type the name of a person or place that you know is in your database. It will not show you pictures from that place or location.
Location sometimes work because the smart search might take some hints (for example, if you type "Japan" you may get some pictures of Japan in your library), but people do not work at all.

**Reply by bo0tzz** [reply]
If you just type in the search bar that goes to the smart search. For filtering by people you can use the advanced search filters on the right.

**Reply by kikendo** [reply]
> If you just type in the search bar that goes to the smart search. For filtering by people you can use the advanced search filters on the right.

that’s not at all what I meant, can you please not close topics which you have not resolved? if you don’t understand a request that is ok but leave it alone. 

The idea is to type a name in the search bar and hits that match that person (OR PLACE)will show up. Just like any other photo search works (like on iOS). I know how to use the system the way it is now. I don’t think it’s a good way especially for precise locations.

**Reply by bo0tzz** [reply]
No need to get crass with me. I understood your request as being for something that is already implemented and so I closed it, simple as that. 

The main search bar is currently used for the smart search, and combining those results with results for people, albums, locations, etc is very hard, so we decided against supporting that for the time being. As such I will leave this feature request closed.
